test/e2e/testnet/node.go

```diff
@@ -199,11 +199,15 @@ func (n *Node) Init(genesis *types.GenesisDoc, peers []string, configOptions ...
 		return fmt.Errorf("writing address book: %w", err)
 	}
 
+	if err := n.Instance.Commit(); err != nil {
+		return fmt.Errorf("committing instance: %w", err)
+	}
+
 	if err := n.Instance.AddFolder(nodeDir, remoteRootDir, "10001:10001"); err != nil {
 		return fmt.Errorf("copying over node %s directory: %w", n.Name, err)
 	}
 
-	return n.Instance.Commit()
+	return nil
```

Comment:
Swapping the order of these two function calls makes the tests fail, in specific, I ran `E2ESimple`.  Below is the log of the first validator which constantly fails to start:
```
2024-05-21 15:01:07.141 | cp: can't stat '/home/celestia/.celestia-app/*': No such file or directory |  
-- | -- | --
  |   | 2024-05-21 15:00:13.162 | cp: can't stat '/home/celestia/.celestia-app/*': No such file or directory |  
  |   | 2024-05-21 14:59:44.139 | cp: can't stat '/home/celestia/.celestia-app/*': No such file or directory |  
  |   | 2024-05-21 14:59:30.184 | cp: can't stat '/home/celestia/.celestia-app/*': No such file or directory
```

Reply:
interesting, it was still passing for me and also in CI. but apparently this performance improvement is only available in version 13.3